src/dense.jl

```diff
@@ -1,193 +1,207 @@
 # A higher level stepper, defined as a wrapper around another stepper.
 
-#TODO: how about having an DenseStepper <: AbstractWrapper <: AbstractStepper?
-immutable DenseStepper <: AbstractStepper
-    solver::Solver
+"""
+
+Dense output options:
+
+- tspan    ::Vector{T}  output times
+- points   ::Symbol which points are returned: `:specified` only the
+  ones in tspan or `:all` which includes also the step-points of the solver.
+- stopevent  Stop integration at a zero of this function
+- roottol    TODO
+
+"""
+
+immutable DenseOptions{T<:Number} <: Options{T}
+    tspan    ::Vector{T}
+    tstop    ::T
+    # points   ::Symbol
```

Comment:
`:all` just means that it should return both the requested outputs and the times the steps were taken.  So a dense output should return those.